config/aws/group_vars/k8s-cluster/ck8s-k8s-cluster-aws.yaml

```diff
@@ -0,0 +1 @@
+etcd_kubeadm_enabled: false
```

Comment:
Is this supposed to be `false` when we have it set to `true` in the default and gcp flavors? Seems very dangerous to me to make them different! [This issue](https://github.com/kubernetes-sigs/kubespray/issues/6687) also seem to indicate that kubeadm managed etcd will be the default in the future.

Reply:
This is based on the AWS guide on compliantkubernetes.io and in turn the AWS terraform script in kubespray. Which creates separate etcd nodes. So either we update the terraform scripts or leave it like this.

Reply:
Ok, then I guess it will have to be this way at least for this PR. I'm uncomfortable having different settings for this in production environments but we are not quite there yet for AWS, right?

Reply:
Yea, I wouldn't say it is